vagrant/do_vagrant_cbs.sh

```diff
@@ -24,24 +25,45 @@ build_vagrant_image()
   if [ ! -t 1 ]; then WAIT="--wait"; fi
 
   EL_MAJOR=$1
-  koji -p cbs image-build \
-    centos-${EL_MAJOR} 1  cloudinstance${EL_MAJOR}-common-el${EL_MAJOR} \
-    http://mirror.centos.org/centos/${EL_MAJOR}/os/x86_64/ x86_64 \
-    --release=1 \
-    --distro RHEL-${EL_MAJOR}.0 \
-    --ksver RHEL${EL_MAJOR} \
-    --kickstart=${KS_DIR}/centos${EL_MAJOR}.ks \
-    --format=vagrant-libvirt \
-    --format=vagrant-virtualbox \
-    --format=vagrant-vmware-fusion \
-    --format=vagrant-hyperv \
-    --factory-parameter fusion_scsi_controller_type pvscsi \
-    --ova-option vagrant_sync_directory=/vagrant \
-    --repo http://mirror.centos.org/centos/${EL_MAJOR}/extras/x86_64/\
-    --repo http://mirror.centos.org/centos/${EL_MAJOR}/updates/x86_64/\
-    --scratch \
-    ${WAIT:-"--nowait"} \
-    --disk-size=40
+  if [ "${EL_MAJOR}" -eq 8 ]; then
+    koji -p cbs image-build \
+      centos-${EL_MAJOR} 1  cloudinstance${EL_MAJOR}-common-el${EL_MAJOR} \
+      http://mirror.centos.org/centos/${EL_MAJOR}/BaseOS/x86_64/ x86_64 \
+      --release=1 \
+      --distro RHEL-${EL_MAJOR}.0 \
+      --ksver RHEL${EL_MAJOR} \
+      --kickstart=${KS_DIR}/centos${EL_MAJOR}.ks \
+      --format=vagrant-libvirt \
+      --format=vagrant-virtualbox \
+      --format=vagrant-vmware-fusion \
+      --format=vagrant-hyperv \
+      --factory-parameter fusion_scsi_controller_type pvscsi \
+      --ova-option vagrant_sync_directory=/vagrant \
+      --repo http://mirror.centos.org/centos/${EL_MAJOR}/extras/x86_64/os/\
+      --repo http://mirror.centos.org/centos/${EL_MAJOR}/AppStream/x86_64/os/\
+      --scratch \
+      ${WAIT:-"--nowait"} \
+      --disk-size=40
```

Comment:
The command is the same as below, except for the paths to the repos; I would prefer to place such differences in variables and call the command just once, this would save us from copying half of page of code.